From d5493386a107e06329496bc09b82ba5d943c8372 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Wed, 21 Nov 2018 16:41:29 +0200
Subject: [PATCH 5179/5242] MLK-20328-7: ARM64: dts: mx8mq-evk: adjust format,
 add 2 pins masks

commit  43b6e97b66d00bbbab9e20e70a9bd7b25060f6f2 from
https://source.codeaurora.org/external/imx/linux-imx.git

a) Default value must be prefixed by "0".
b) Map 2 pins case to rx=0xff tx=0x11.

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
(cherry picked from commit cb4d69f098ee003edc2dcfec7075b043a78345e6)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 4c0c061..a9d93cb 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -814,7 +814,7 @@
 		<&clk IMX8MQ_AUDIO_PLL2_OUT>;
 	clock-names = "bus", "mclk0", "mclk1", "mclk2", "mclk3", "pll8k", "pll11k";
 	fsl,sai-multi-lane;
-	fsl,dataline,dsd = <0xff 0x11>;
+	fsl,dataline,dsd = <0 0xff 0xff 2 0xff 0x11>;
 	dmas = <&sdma2 8 26 0>, <&sdma2 9 26 0>;
 	status = "okay";
 };
-- 
1.7.9.5

